runtime/lua/vim/treesitter/dev.lua

```diff
@@ -1,16 +1,16 @@
 local api = vim.api
 
----@class TSPlaygroundModule
+---@class TSDevModule
 local M = {}
 
----@class TSPlayground
+---@class TSView
```

Comment:
```suggestion
---@class TSTreeView
```
This is specifically a view of the LanguageTree, not a generic view (e.g. of queries).